styles.css

```
@@ -0,0 +1,95 @@
+
+
+body {
+  background-color: #333;
+  color: #7fff00;
+  font-family: "Lucida Console", Monaco, monospace;
+}
+
+a {
+  color: #66cd00;
+}
+
+a:hover {
+  color: #ffd700;
+}
+
+nav>ul {
+  width: 25em;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+li {
+  list-style: none;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  padding: 0.5em 1em;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #000;
+}
+
+#wrapper {
+  width: 70%;
+  margin: 5em auto;
+}
+
+h1 {
+  text-align: center;
+}
+
+section {
+  margin: 2em auto;
+}
+
```

If you want to center an inline element, like the iframe, you can use this:
```suggestion
section > iframe {
  display: block;
  margin: 0 auto;
}
```
It transforms the inline element in a block element and then you can use any properties that block elements accept.